features/step_definitions/create_menu_page.rb

```diff
@@ -16,6 +16,10 @@
       path = new_user_registration_path
     when 'restaurants' then
       path = new_restaurant_path
+    when 'edit' then
```

Comment:
This is not really necessary unless you make it a generic case that can be used to return edit page of a given resource. You already have a step that visits the edit page for restaurant, you can remove this.